diff --git a/lib/fx2.mk b/lib/fx2.mk
index 501040f..359a698 100644
--- a/lib/fx2.mk
+++ b/lib/fx2.mk
@@ -41,15 +41,15 @@ VID?=0x04b4
 PID?=0x8613

 INCLUDES?=""
-DSCR_AREA?=-Wl"-b DSCR_AREA=0x3e00"
-INT2JT?=-Wl"-b INT2JT=0x3f00"
+DSCR_AREA?=-Wl"-b DSCR_AREA=0x1e00"
+INT2JT?=-Wl"-b INT2JT=0x1f00"
 CC=sdcc
 # these are pretty good settings for most firmwares.
 # Have to be careful with memory locations for
 # firmwares that require more xram etc.
-CODE_SIZE?=--code-size 0x3c00
+CODE_SIZE?=--code-size 0x1c00
 XRAM_SIZE?=--xram-size 0x0200
-XRAM_LOC?=--xram-loc 0x3c00
+XRAM_LOC?=--xram-loc 0x1c00
 BUILDDIR?=build

 FX2LIBDIR?=$(dir $(lastword $(MAKEFILE_LIST)))../
